### PR TITLE
[osgi] Solutions for https://github.com/eclipse/microprofile/issues/33

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

File: api/bnd.bnd [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+
 Import-Package: \
-    javax.enterprise.util;version="[1.1,3)", \
+    javax.enterprise.util;-remove-attribute:=version,\
+    javax.interceptor;-remove-attribute:=version,\
     *
+
+Require-Capability: \
+    osgi.contract;\
+        osgi.contract=JavaCDI;\
```
Comment (Emily-Jiang) [inline api/bnd.bnd]: Can you remind me which bundle provide this capability?

**Comment by rotty3000** [comment]
The bundle that contains the CDI API should provide it. Or a fragment to it.

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

**Comment by carlosdlr** [comment]
This was close as won't do

**Pull request closed without merging**